### ISSUE TITLE
Update Accessing deployments endpoint in demo notebook 

### ISSUE THREAD

**Issue by kernel-loophole** [mozilla-ai/lumigator#297]
While trying to list all available deployments using the `ld.get_deployments()` [function,](https://github.com/mozilla-ai/lumigator/blob/main/notebooks/lumigator_demo.py#L311) a 404 Client Error occurred. The request failed for the `URL: http://0.0.0.0:8000/api/v1/ground-truth/deployments/`.
Possible Causes:
- The endpoint URL might be incorrect.
- The deployment server may not be running or listening on the specified port.
- The base API path or version (/api/v1/ground-truth/) might be outdated or not configured properly.
![Screenshot from 2024-10-21 08-17-38](https://github.com/user-attachments/assets/5559de86-e314-46b2-bb25-788e821987f3)

**Comment by javiermtorres** [comment]
Could you try `http://127.0.0.1:8000/api/v1/ground-truth/deployments/` instead? Usually `0.0.0.0` indicates the kernel that a server wants to attach to all possible interfaces (with their IPs). I'm not sure how a _request_ to `0.0.0.0` is interpreted.

**Comment by kernel-loophole** [comment]
sure

**Comment by javiermtorres** [comment]
Hi @kernel-loophole I was puzzled with the source IP, so I didn't check that you're addressing the `deployments` endpoint, which is no longer available due to changes in internal architecture. Please give us a few days to provide an alternative.

**Comment by kernel-loophole** [comment]
@javiermtorres yeah sure ,let me know if you have anything for me  where i can contribute .

**Comment by kernel-loophole** [comment]
@javiermtorres i think this issue is now fixed #345
